docs/app/shared/landing-shared.module.ts

```diff
@@ -31,6 +35,7 @@ const pipes = [
   ],
   entryComponents: [
     DownloadFormComponent,
+    PremiumFormComponent,
   ],
```

Comment:
You don't need to add dynamic components to `entryComponents` starting from Angular 9 or 10.